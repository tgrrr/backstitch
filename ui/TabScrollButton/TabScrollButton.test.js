import React from 'react'
import renderer from 'react-test-renderer'
import { TabScrollButton } from '.'

it('TabScrollButton: default', () => {
    const component = renderer.create(
        <TabScrollButton>
            <>
                TabScrollButton
            </>
        </TabScrollButton>
    )
  const tree = component.toJSON()
  expect(tree).toMatchSnapshot()
})
