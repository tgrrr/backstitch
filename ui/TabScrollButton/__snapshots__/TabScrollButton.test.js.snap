// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TabScrollButton: default 1`] = `
<div
  className="sc-AxjAm StDqN"
>
  <div
    className="Tabscrollbutton "
    data-testid="Tabscrollbutton"
  >
    <div
      aria-disabled={false}
      className="MuiButtonBase-root MuiTabScrollButton-root MuiTabScrollButton-vertical"
      onBlur={[Function]}
      onDragLeave={[Function]}
      onFocus={[Function]}
      onKeyDown={[Function]}
      onKeyUp={[Function]}
      onMouseDown={[Function]}
      onMouseLeave={[Function]}
      onMouseUp={[Function]}
      onTouchEnd={[Function]}
      onTouchMove={[Function]}
      onTouchStart={[Function]}
      role={null}
      tabIndex={null}
    >
      <svg
        aria-hidden={true}
        className="MuiSvgIcon-root MuiSvgIcon-fontSizeSmall"
        focusable="false"
        viewBox="0 0 24 24"
      >
        <path
          d="M8.59 16.34l4.58-4.59-4.58-4.59L10 5.75l6 6-6 6z"
        />
      </svg>
    </div>
  </div>
</div>
`;
